packages/scripts/index.js

```diff
@@ -30,21 +33,50 @@ const getOutputDir = (output) => {
   return outputDirMatch.length > 1 ? outputDirMatch[1] : output;
 };
 
+// Runs typescript compiler for type checking and emitting declarations in a separate process to run in parallel with
+// the bundler.
+const runTsc = (options) => {
+  const tscBin = path.resolve(`./node_modules/.bin/tsc`);
+  return exec(`${tscBin} --emitDeclarationOnly`)
+    .then(() => ora().succeed(chalk.green.bold('type checking done')))
+    .catch(({ stderr, stdout }) => {
```

Comment:
If I'm reading this right, it's impossible to fail due to typescript compilation errors?  That's obviously OK for dev mode, but we want to fail when building the prod bundle.

Reply:
Good call. Going to fix that.